.github/workflows/aks-letsencrypt.yml

```diff
@@ -148,7 +148,6 @@ jobs:
           # available version is the one provided by GitHub. So calling a bash script from a
           # Makefile lead to this error: 'Windows Subsystem for Linux has no installed distributions.'
           # The workaround is to execute the bash script directly
-          ./scripts/setup-chart-museum.sh
```

Comment:
The comments before can be removed here, there were for chartmuseum ;-)

Reply:
Ah, I haven't read them throughly :sweat_smile: Thanks